### DISCUSSION TITLE
guide/drivers/baidu

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#127] [Announcements]
# guide/drivers/baidu

Due to the limitation of Baidu Disk API, downloading files larger than about 20M needs to carry the header: 

https://alist.nn.ci/guide/drivers/baidu.html

**Comment by GaryRandom** [comment]
I run Alist docker in my NAS. I want to know **how to download files directly in my NAS**, instead of downloading files in the laptop used for browsering.

**Comment by kokojacket** [comment]
挂载百度网盘，可下载文件到PC，但上传文件提示：找不到该项目

**Comment by slycmder** [comment]
我有两个百度网盘，都是VIP，使用第一个网盘挂载后，下载文件的速度很快，但是切换第二个网盘，速度就比较慢了，重新切换回第一个网盘后速度还是很快。我在其他电脑上挂载第二个百度网盘，速度也会很快。怀疑是因为同一个client id使用不同的token导致的，各位有什么方法刷新client id，或者切换百度网盘而不限速的吗？
